gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/store/acceptOwnership.ts

```diff
@@ -0,0 +1,49 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { SolanaCommand, TransactionResponse } from '@chainlink/gauntlet-solana'
+import { PublicKey } from '@solana/web3.js'
+import { CONTRACT_LIST, getContract } from '../../../lib/contracts'
+
+export default class AcceptOwnership extends SolanaCommand {
+  static id = 'store:accept_ownership'
+  static category = CONTRACT_LIST.STORE
+
+  static examples = ['yarn gauntlet store:accept_ownership --network=devnet --state=[STORE_STATE]']
+
+  constructor(flags, args) {
+    super(flags, args)
+
+    this.require(!!this.flags.state, 'Please provide flags with "state"')
+  }
+
+  execute = async () => {
+    const store = getContract(CONTRACT_LIST.STORE, '')
+    const storeAddress = store.programId.toString()
+    const storeProgram = this.loadProgram(store.idl, storeAddress)
+
+    const owner = this.wallet.payer
+
+    const storeState = new PublicKey(this.flags.state)
+
+    await prompt(`Accepting ownership of store state (${storeState.toString()}). Continue?`)
```

Comment:
small nit pick, but would be nice to validate that the proposed owner onchain is the same as the proposed owner. but that can be tackled at a later point too 😄

Reply:
The tx will revert if it's not like that, but is a nice to have. Leaving it for next iteration